<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1424</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>The Commander</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/command.png</normaloff>../images/command.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>1400</width>
        <height>588</height>
       </size>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="3">
           <widget class="QPushButton" name="move_down_button_1">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>..</string>
            </property>
            <property name="icon">
             <iconset theme="go-down">
              <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="home_button_1">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="directory_line_1">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>/Users/alanford/Desktop</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="terminal_button_1">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Terminal</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QTreeView" name="treeView_1">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="kind_1">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>kind</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="date_1">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="hide_unhide_button_1">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Show/Hide tree</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="browse_1">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="match_button_1">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="size_1">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QPushButton" name="hide_unhide_button_2">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Show/Hide tree</string>
            </property>
           </widget>
          </item>
          <item row="2" column="6">
           <widget class="QTreeView" name="treeView_3">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="9">
           <widget class="QPushButton" name="move_down_button_2">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>..</string>
            </property>
            <property name="icon">
             <iconset theme="go-down">
              <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="10">
           <widget class="QPushButton" name="match_button_2">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QLineEdit" name="directory_line_2">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>/Users/alanford/Desktop</string>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <widget class="QPushButton" name="browse_2">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item row="0" column="11">
           <widget class="QPushButton" name="home_button_2">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item row="1" column="8">
           <widget class="QPushButton" name="terminal_button_2">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Terminal</string>
            </property>
           </widget>
          </item>
          <item row="1" column="9">
           <widget class="QPushButton" name="size_2">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="10">
           <widget class="QPushButton" name="kind_2">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>kind</string>
            </property>
           </widget>
          </item>
          <item row="1" column="11">
           <widget class="QPushButton" name="date_2">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item row="2" column="7" colspan="5">
           <widget class="QTreeView" name="treeView_4">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="5">
           <widget class="QTreeView" name="treeView_2">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="move_button">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Move</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="copy_button">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Copy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="compare_button">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Compare</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_button">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="permission_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>change 
 permissions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_9">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_16">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_15">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_10">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_12">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_14">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_13">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_11">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1424</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>browse_1</sender>
   <signal>clicked(bool)</signal>
   <receiver>directory_line_1</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browse_2</sender>
   <signal>pressed()</signal>
   <receiver>directory_line_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1195</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browse_1</sender>
   <signal>pressed()</signal>
   <receiver>directory_line_1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browse_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>directory_line_2</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1195</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
